middleware/limiter/limiter_fixed.go

```diff
@@ -27,8 +26,11 @@ func (FixedWindow) New(cfg Config) fiber.Handler {
 
 	// Return new handler
 	return func(c fiber.Ctx) error {
-		// Don't execute middleware if Next returns true
-		if cfg.Next != nil && cfg.Next(c) {
+		// Generate max from generator, if no generator was provided the default value returned is 5
+		max := cfg.MaxCalculator(c)
```

Comment:
How much does this affect the benchmark?

Reply:
Hmmmm, I'll make a comparison for you to check, but I don't think it will be that much slower (if it's slower at all)

Reply:
Results before the change:
```
goos: linux
goarch: amd64
pkg: github.com/gofiber/fiber/v3/middleware/limiter
cpu: Intel(R) Core(TM) i7-10875H CPU @ 2.30GHz
Benchmark_Limiter_Custom_Store-16        3361042               361.7 ns/op            72 B/op          2 allocs/op
Benchmark_Limiter-16                     4769715               258.0 ns/op             8 B/op          1 allocs/op
Benchmark_MarshalMsgitem-16             30891702                32.66 ns/op           64 B/op          1 allocs/op
Benchmark_AppendMsgitem-16              168474463                7.242 ns/op    3590.40 MB/s           0 B/op          0 allocs/op
Benchmark_Unmarshalitem-16              44539614                26.27 ns/op      989.83 MB/s           0 B/op          0 allocs/op
```

Results after the change:
```
goos: linux
goarch: amd64
pkg: github.com/gofiber/fiber/v3/middleware/limiter
cpu: Intel(R) Core(TM) i7-10875H CPU @ 2.30GHz
Benchmark_Limiter_Custom_Store-16        3335551               360.5 ns/op            72 B/op          2 allocs/op
Benchmark_Limiter-16                     4715961               254.1 ns/op             8 B/op          1 allocs/op
Benchmark_MarshalMsgitem-16             36100575                31.82 ns/op           64 B/op          1 allocs/op
Benchmark_AppendMsgitem-16              154027914                8.133 ns/op    3196.81 MB/s           0 B/op          0 allocs/op
Benchmark_Unmarshalitem-16              42412798                24.97 ns/op     1041.17 MB/s           0 B/op          0 allocs/op
```

Reply:
The difference seems minimal

Reply:
Another result before the changes including the logic in limiter_sliding.go:
```
goos: linux
goarch: amd64
pkg: github.com/gofiber/fiber/v3/middleware/limiter
cpu: Intel(R) Core(TM) i7-10875H CPU @ 2.30GHz
Benchmark_Limiter_Custom_Store-16        3495290               356.1 ns/op            72 B/op          2 allocs/op
Benchmark_Limiter-16                     4842919               240.1 ns/op             8 B/op          1 allocs/op
Benchmark_MarshalMsgitem-16             35154388                36.24 ns/op           64 B/op          1 allocs/op
Benchmark_AppendMsgitem-16              173502765                6.867 ns/op    3786.38 MB/s           0 B/op          0 allocs/op
Benchmark_Unmarshalitem-16              47368640                25.09 ns/op     1036.46 MB/s           0 B/op          0 allocs/op
```